client/landing/gutenboarding/components/close-button/index.tsx

```diff
@@ -5,14 +5,14 @@ import React from 'react';
 import { Button } from '@wordpress/components';
 import { useI18n } from '@automattic/react-i18n';
 
-interface CloseButtonProps {
+interface CloseButtonProps extends Button.ButtonProps {
 	onClose: () => void;
 }
 
-const CloseButton = ( { onClose }: CloseButtonProps ) => {
+const CloseButton = ( { onClose, ...buttonProps }: CloseButtonProps ) => {
 	const { __ } = useI18n();
 	return (
-		<Button onClick={ onClose } label={ __( 'Close dialog' ) }>
+		<Button onClick={ onClose } label={ __( 'Close dialog' ) } { ...buttonProps }>
 			<svg
```

Comment:
We should be using `SVG` from `@wordpress/components` (or `@wordpress/primitives`?) here.

Reply:
Other places are importing from `@wordpress/components`. Just gonna follow. This commit imports the `SVG` element. https://github.com/Automattic/wp-calypso/pull/41007/commits/b5aabe8e37b99c5ff2880ba450949002f1f4c894